.github/workflows/linters.yml

```diff
@@ -27,3 +27,17 @@ jobs:
 
     - name: Run rubocop
       run: bundle exec rubocop --parallel
+
+
+    - name: Slack notification
+      if: success() || failure()
+      uses: rtCamp/action-slack-notify@v2
+      env:
+         SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+         SLACK_CHANNEL: "ci-${{ github.event.repository.name }}"
+         SLACK_COLOR: ${{ job.status }} # automatic colour based on job status
+         SLACK_ICON_EMOJI: ':sharesight_cat:'
```

Comment:
I wonder if it's better to use a `:github:` emoji here (we need to add this to Slack then) to indicate where the message comes from.

Reply:
We have the :octocat: `:octocat:` emoji already I believe so could probably even use that?

Reply:
💯